src/material/sidenav/sidenav.md

```diff
@@ -209,6 +209,11 @@ Similarly, the `<mat-sidenav-content>` should be given a role based on what it c
 represents the primary content of the page, it may make sense to mark it `role="main"`. If no more
 specific role makes sense, `role="region"` is again a good fallback.
 
+#### Focus management
+To enable auto focus within `<mat-drawer>` component, you need to set `[autoFocus]=true`. 
+
+By default the first tabbable element will recieve focus upon open. This can be configured by setting the `cdkFocusInitial` attribute on another focusable element.
```

Comment:
"This can be configured by setting the `cdkFocusInitial` attribute on another focusable element." -> "If you want a different element to be focused, you can set the `cdkFocusInitial` attribute on it."

Reply:
yes it's definitely better explanation, I'll update it